src/VisualStudio/Core/Def/EditorConfigSettings/DataProvider/CodeStyle/CommonCodeStyleSettingsProvider.cs

```diff
@@ -187,5 +196,23 @@ private IEnumerable<CodeStyleSetting> GetParameterCodeStyleOptions(AnalyzerConfi
                 editorConfigOptions: options,
                 visualStudioOptions: visualStudioOptions, updater: updater, fileName: FileName);
         }
+
+        private IEnumerable<CodeStyleSetting> GetExperimentalCodeStyleOptions(AnalyzerConfigOptions options, OptionSet visualStudioOptions, OptionUpdater updater)
```

Comment:
Are these options going to be visibly different to a user?

Reply:
No they are not, they are written out with `_experimental` to the file though